RELEASE-NOTES.txt

```diff
@@ -1,6 +1,13 @@
+3.6
```

Comment:
This is kind of unexpected merging the `release/3.5` branch. 
Do you know why it's here?

Reply:
Yeah – https://github.com/woocommerce/woocommerce-ios/commit/238063d64347cb110407e0d092192f2bb2055d53 was merged prior to the last code freeze (on Jan 22, code freeze was Jan 27), and introduced the `3.6` notes early.

Reply:
So the feature was under a feature flag and it's been released in 3.6?

Reply:
It wasn't under a feature flag, but it _was_ released in 3.6 and is in the 3.6 release notes at `p2JRYi-4qp` :)

Reply:
👍